Fix issue #131

jtamagnan opened the pull request:
Issue #131 would occur because the function was receiving a raw prefix
arg instead of a number. This commit fixes that.

jtamagnan commented:
@magnars This commit fixes issue #131 as such that issue should be closed. The issue arose when `C-u mc/insert-numbers` was called instead of `C-u 4 mc/insert-numbers`. Now both versions work

magnars commented:
Excellent! Would you mind taking a look at the failing tests? https://travis-ci.org/magnars/multiple-cursors.el/builds/126212853

jtamagnan commented:
I fixed the bug that existed previously (I had taken for granted that the default for `(interactive "p")` was going to be nil or 0 when it is infact 1! I also added some extra test cases

purcell commented:
NIce - thanks @jtamagnan!
